packages/app/screens/swipe-list.tsx

```diff
@@ -89,9 +91,10 @@ const ProfileSwipeList = () => {
   );
   const { bottom: safeAreaBottom } = useSafeAreaInsets();
 
-  const initialScrollIndex = data?.findIndex(
-    (item) => item.nft_id == initialScrollItemId
-  );
+  const initialScrollIndex = useMemo(() => {
+    const index = data?.findIndex((item) => item.nft_id == initialScrollItemId);
```

Comment:
how you handle data if undefined? Or initialScrollItemId was tampered with?
The optional chain would fail

Reply:
if data is undefined `initialScrollIndex` would be undefined and won't be passed in the list so it'll default to 0 in swipe-list and if `initialScrollItemId` is weird then it'll be -1 and so will default to 0. seems okay to me.

Also when data is 0 swipe-list doesn't render anything

Reply:
But if data is undefined, findIndex would fail with "data is undefined".
It should be 

```
const index = data ? data.findIndex.. : 0
```

Reply:
Or will SWR make sure to always return an empty array?

Reply:
oh try this. 
x = undefined; x?.findIndex(); 
this won't fail but will be returned as undefined. The only time it can fail is if data is not an array and not undefined. SWR will return undefined or an array.

Reply:
LGTM

Reply:
I forgot about the hook definition!